constraint/pkg/client/client.go

```diff
@@ -767,6 +786,36 @@ func (c *Client) Dump(ctx context.Context) (string, error) {
 	return dumpBuilder.String(), nil
 }
 
+func (c *Client) GetDescriptionForStat(source instrumentation.Source, statName string) string {
+	if source.Type != instrumentation.EngineSourceType {
+		// only handle engine source for now
+		return instrumentation.UnknownDescription
+	}
+
+	// this is written in a general form
+	// but it only works for rego drivers for now.
+	for dName, d := range c.drivers {
```

Comment:
This should not require a for loop. Drivers are indexed by name, so there can be a direct lookup.